flybywiresim/a32nx
# feat(model): custom gravity gear extension model

## Summary of Changes

- Added custom animated gravity gear extension model
- Adjusted phone cord

## Screenshots (if necessary)


![image](https://user-images.githubusercontent.com/16512508/181081434-160dbe90-22c5-40a1-a7f2-b5bcd0bdf0f8.png)
![image](https://user-images.githubusercontent.com/16512508/181081568-08daa440-ca52-4cf9-a46a-54b56b33f422.png)
![image](https://user-images.githubusercontent.com/16512508/181081608-98bb2307-6e04-4150-bb8a-97bb3a015e61.png)

## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

- Make sure there are no issues with the model/animation
- Make sure the gravity gear extension still works


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
